src/fsharp/TypeChecker.fs

```diff
@@ -14195,14 +14195,15 @@ module EstablishTypeDefinitionCores = begin
             let writeFakeRecordFieldsToSink (fields:RecdField list) =
                 let nenv = envinner.NameEnv
                 // Record fields should be visible from IntelliSense, so add fake names for them (similarly to "let a = ..")
-                for fspec in (fields |> List.filter (fun fspec -> not fspec.IsCompilerGenerated)) do
-                    let info = RecdFieldInfo(thisTyInst, thisTyconRef.MakeNestedRecdFieldRef fspec)
-                    let nenv' = AddFakeNameToNameEnv fspec.Name nenv (Item.RecdField info) 
-                    // Name resolution gives better info for tooltips
-                    let item = FreshenRecdFieldRef cenv.nameResolver m (thisTyconRef.MakeNestedRecdFieldRef fspec)
-                    CallNameResolutionSink cenv.tcSink (fspec.Range,nenv,item,item,ItemOccurence.Binding,envinner.DisplayEnv,ad)
-                    // Environment is needed for completions
-                    CallEnvSink cenv.tcSink (fspec.Range, nenv', ad)
+                for fspec in fields do
+                    if not fspec.IsCompilerGenerated then
+                        let info = RecdFieldInfo(thisTyInst, thisTyconRef.MakeNestedRecdFieldRef fspec)
+                        let nenv' = AddFakeNameToNameEnv fspec.Name nenv (Item.RecdField info) 
+                        // Name resolution gives better info for tooltips
+                        let item = FreshenRecdFieldRef cenv.nameResolver m (thisTyconRef.MakeNestedRecdFieldRef fspec)
+                        CallNameResolutionSink cenv.tcSink (fspec.Range,nenv,item,item,ItemOccurence.Binding,envinner.DisplayEnv,ad)
```

Comment:
@vasily-kirichenko I assume there are several reasons:

* code review on the compiler code is very sensitive, if it can be done easily "inline" from github because the diff shows minimally (sadly github isn't that clever for whitespace/indent change only) this saves a lot of time
* to improve codebase readability, it is better to do as small changes as possible when dealing with readability only, and keeping an eye on making the diff of each change easily reviewable more than reaching "most readable" state as soon as possible
* @dsyme has to merge changes to fsharp/fsharp and fsharp/FSharp.Compiler.Services repositories, despite I don't know how smooth that tend to go, I assume he still has to pretty much review each piece which is different, but I think in this usecase he does so with another diff tool than what we see on github pages, but still less diff when there is 0 change in the code is better for him to not spend too much time and keep the motivation to do this tedious work (that nobody else ever did, at least not at the scale he does) so it can happen as frequently and as safely as needed


My feeling is that cleaning the codebase in terms of style would help newcommers in the long run, but this incur overhead for the people currently dealing with 99% of the work on compiler today. I don't know how this shift will gradually occur but it is an interesting matter :smile: